molecule/ubuntu1804/molecule.yml

```diff
@@ -0,0 +1,39 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+platforms:
+  - name: nginx-default
+    image: leandelivery/docker-systemd:ubuntu-18.04
```

Comment:
I haven't investigated but I suspect this means there is no official Ubuntu Docker image with systemd support?

Reply:
No official images, this one was mentioned on https://github.com/ansible/molecule/issues/1768